src/bundle/Resources/views/content/tabs/urls.html.twig

```diff
@@ -0,0 +1,75 @@
+{% trans_default_domain "locationview" %}
+
+<div class="ez-list-toolbar">
+    <h2 class="ez-list-toolbar-label">{{ 'locationview.urls.custom.title'|trans({'%contentName%' : ez_content_name(content)})|desc('Custom URL aliases for %contentName%') }}</h2>
+</div>
+
+{% if customUrls %}
+    <table class="ez-table-data">
+        <thead>
+        <tr>
+            <th></th>
+            <th>{{ 'locationview.urls.url'|trans|desc('URL') }}</th>
+            <th>{{ 'locationview.urls.language'|trans|desc('Language') }}</th>
+            <th>{{ 'locationview.urls.type'|trans|desc('Type') }}</th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for customUrl in customUrls %}
+            <tr>
+                <td></td>
+                <td>
+                    <a href="{{ customUrl.path }}">{{ customUrl.path }}</a>
```

Comment:
according to description in https://jira.ez.no/browse/EZP-27448, this is not supposed to be a link (which leads to a different siteaccess btw)